src/SDKs/Network/Management.Network/Generated/INetworkInterfaceLoadBalancersOperations.cs

```diff
@@ -19,15 +19,18 @@ namespace Microsoft.Azure.Management.Network
     using System.Threading.Tasks;
 
     /// <summary>
-    /// AvailablePrivateAccessServicesOperations operations.
+    /// NetworkInterfaceLoadBalancersOperations operations.
     /// </summary>
-    public partial interface IAvailablePrivateAccessServicesOperations
+    public partial interface INetworkInterfaceLoadBalancersOperations
```

Comment:
> Does it have to be plural (LoadBalancerS)?
I guess. I named operationIds in Swagger based on existing list operations which were in plural form (e.g. `NetworkInterfaceIPConfigurations`). Makes sense since they suppose to list all resources of a certain type or get a specific one.

> Also why the interface is renamed differently than the interface implementing class
I think you confusing two separate entities. There are both NetworkInterfaceLoadBalancersOperations (load balancers referenced by network intefrace) and LoadBalancerNetworkInterfacesOperations (the other way around).